### ISSUE TITLE
rGuiStyler on itch.io export as code gives an invalid font image

### ISSUE THREAD

**Issue by schveiguy** [raysan5/rguistyler#10]
I have noticed when trying to use the online rGuiStyler, and I export a style as a .h file, the file does not load properly for my code, the screen is very much garbage.

After a lot of investigation, I have narrowed it down to the image format being used for the font image.

When I run rGuiStyler locally, I get this as the initialization of the font image:

```c
    // Custom font loading
    // NOTE: Compressed font image data (DEFLATE), it requires DecompressData() function
    int genesisFontDataSize = 0;
    unsigned char *data = DecompressData(genesisFontData, GENESIS_STYLE_FONT_ATLAS_COMP_SIZE, &genesisFontDataSize);
    Image imFont = { data, 512, 256, 1, 2 };
```
However, when using the itch.io download, the last line comes out like this:
```c
    Image imFont = { data, 512, 256, 1, 7 };
```
Note the difference in the last number. This is the format.

The 2 corresponds to `PIXELFORMAT_UNCOMPRESSED_GRAY_ALPHA`, whereas the 7 corresponds to `PIXELFORMAT_UNCOMPRESSED_R8G8B8A8`

The export code loads the image here: https://github.com/raysan5/rguistyler/blob/9d1fea319b60a6e6119c58721a92aaa3b1265701/src/rguistyler.c#L2537

Note the check right afterwards to validate the image format is correct, but all it does is print a warning. I don't know where this warning goes on the itch.io page, so I'm not sure if it's visible anywhere.

Trying to understand why it's 7 and not 2, I looked at the code for `LoadImageFromTexture`, and noticed this override:
https://github.com/raysan5/raylib/blob/84737a9fc19669b04ce86eb458033b81089bbddc/src/rtextures.c#L577-L582

Clearly, this is where the 7 could be set. I don't know if this code is active for WASM or not.

Not sure what the fix should be. Maybe override the image again with the texture format? I'm not sure why this override is there, but it's pretty old. Here is the commit: https://github.com/raysan5/raylib/commit/e614942a2d52ea79c3a7f0fda4f0357035a6cc53
